build.gradle

```diff
@@ -38,7 +38,8 @@ buildscript {
         fineractPublishProjects = subprojects.findAll{
             [
                 'fineract-avro-schemas',
-                'fineract-client'
+                'fineract-client',
+                'fineract-provider'
```

Comment:
The addition causes a failure for the publication for me and for the openMF/fineract repo.